bin/ch-test

```diff
@@ -1138,6 +1162,7 @@ case $phase in
         test_make
         test_build_images
         test_build
+        test_rootemu
```

Comment:
I'd leave it out of build. If you do leave it in, you'll want to figure out what `phase=rootemu` is and whether you need it.